treeherder/model/management/commands/cycle_data.py

```diff
@@ -62,6 +63,11 @@ def handle(self, *args, **options):
                                                 options['sleep_time'])
             self.debug("Deleted {} jobs from {}".format(rs_deleted,
                                                         repository.name))
+            if repository.expire_performance_data:
```

Comment:
We set `expire_performance_data` to `default=True` in `treeherder/perf/models.py`